net/fixed-bytes/src/lib.rs

```diff
@@ -43,6 +49,7 @@ impl Debug for FixedByteError {
 /// Internally this is just a wrapper around [`Bytes`], with the constructors checking that the length is equal to `N`.
 /// This implements [`Deref`] with the target being `[u8; N]`.
 #[derive(Debug, Clone, Eq, PartialEq)]
+#[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
```

Comment:
the `Serialize` is probably ok but the `Deserialize` has to make sure the `ByteArray` is a multiple of `N`